tests/snapshots/src/main/generated/com/airbnb/epoxy/AsyncEpoxyController.java

```diff
@@ -40,8 +40,8 @@ public AsyncEpoxyController() {
    *                    both on the main thread.
    */
   public AsyncEpoxyController(boolean enableAsync) {
-//       ^^^^^^ definition com/airbnb/epoxy/AsyncEpoxyController#`<init>`(+1).
-//                                    ^^^^^^^^^^^ definition local0
+//       ^^^^^^ definition com/airbnb/epoxy/AsyncEpoxyController#`<init>`(+1). public <init>(boolean enableAsync)
```

Comment:
What do you think about showing the class name in this case?
```suggestion
//       ^^^^^^ definition com/airbnb/epoxy/AsyncEpoxyController#`<init>`(+1). public AsyncEpoxyController(boolean enableAsync)
```

Reply:
We can get the symbol owner's descriptor using the new symbol parser